whois-commons/src/test/java/net/ripe/db/whois/common/dao/jdbc/AbstractDatabaseHelperIntegrationTest.java

```diff
@@ -85,5 +87,7 @@ public void setDatabaseHelper(final DatabaseHelper databaseHelper) {
         this.whoisTemplate = databaseHelper.getWhoisTemplate();
         this.internalsTemplate = databaseHelper.getInternalsTemplate();
         this.mailupdatesTemplate = databaseHelper.getMailupdatesTemplate();
+        this.nrtmClientInfoTemplate = databaseHelper.getNrtmClientInfoTemplate();
```

Comment:
We don't need an NRTM client in every integration test, rather I suggest extending the integration tests *only* in the NRTM client code and use there only

Reply:
yes, done